src/AlgebraicGeometry/Schemes/AffineAlgebraicSet/Objects/Methods.jl

```diff
@@ -66,3 +66,84 @@ function Base.show(io::IO, X::AffineAlgebraicSet)
   end
 end
 
+########################################################
+# (2) give back k-points (if finitely many)
+########################################################
+@doc raw"""
+    rational_point_coordinates(I::MPolyIdeal)
+
+Over the base field $k$, return the $k$-coordinates of the point 
+corresponding to a maximal ideal $I \in k[x_1,\dots,x_n]$, which 
+describes a $k$-point. 
```

Comment:
Isn't the first part ("Over the base field $k$") a bit redundant? Perhaps I am missing some subtlety? Also, "point" occurs twice. 

How about this:
```suggestion
Return the $k$-coordinates of the point described by $I \unlhd k[x_1,\dots,x_n]$
assuming that $I$ is a maximal ideal and actually describes a $k$-point, where $k$ is the base field.

```

Reply:
I rephrased it in a shorter way now.